python/setup.py

```diff
@@ -60,6 +60,7 @@
                 '../thirdparty/cutlass',
                 '../thirdparty/cub',
                 '../thirdparty/treelite/include',
+                '../thirdparty/treelite/dmlc-core/include',
```

Comment:
This is probably unnecessary? Note sure where it comes from.

Reply:
This is required for compiling the cython files